okapi/views/devel/cronreport.php

```diff
@@ -18,7 +18,7 @@ public static function call()
         $response->content_type = "text/plain; charset=utf-8";
         ob_start();
 
-        require_once "okapi/cronjobs.php";
+        require_once __DIR__ . '/../../cronjobs.php';
```

Comment:
services a done - views are under construction and in this step i will fix the rest classes.